src/Rendering/VTKJS/Images/applyLookupTable.js

```diff
@@ -27,11 +27,6 @@ function applyLookupTable(context, event) {
 
   const uniqueLabels = Array.from(actorContext.uniqueLabels)
 
-  colorTransferFunction.setMappingRange(
-    uniqueLabels[0],
-    uniqueLabels[uniqueLabels.length - 1]
-  )
-
```

Comment:
Why is this removed?

Reply:
`setMappingRange` is called in call to `applyCategoricalColorToColorTransferFunction` on next line.